lib/src/components/NxTreeView/NxTreeView.scss

```diff
@@ -8,88 +8,92 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+// The caret is taller than it is wide. It is rotated using `transform`, which does not change its effect
+// on the layout of surrounding elements.  However, we _want_ its rotation to affect the layout of surrounding
+// elements such that the title is always 8px from the nearest visible point on the caret. This variable
+// represents the distance necessary to make that happen
+$twisty-proportion-difference: 2px;
+
 /*
   #Tree view
 */
 .nx-tree-view {
-  @include semi-bold();
-
   box-sizing: border-box;
+  margin: $nx-spacing-md 0;
+
+  // Design specifies 300px wide not including the caret and its padding (which are themselves 30px)
   max-width: 330px;
 }
 
 .nx-tree-view--expanded {
-  .nx-tree-view__twisty-icon {
+  .nx-tree-view__twisty {
     transform: rotate(90deg);
+    padding: 0 $twisty-proportion-difference;
   }
 }
 
 .nx-tree-view--collapsed {
   .nx-tree-view__children{
-    max-height: 0;
+    display: none;
   }
 }
 
 .nx-tree-view--disabled {
-  .nx-tree-view__trigger {
-    cursor: default;
-  }
-
-  .nx-tree-view__text, .nx-tree-view__child {
+  .nx-tree-view__trigger, .nx-tree-view__child {
     color: $nx-grey-4;
-  }
-
-  .nx-tree-view__twisty-icon {
-    border-left-color: $nx-grey-4;
+    cursor: default;
   }
 }
 
 .nx-tree-view--empty {
-  .nx-tree-view__trigger, .nx-tree-view__twisty {
+  .nx-tree-view__trigger {
     cursor: default;
   }
 
-  .nx-tree-view__twisty-icon {
-    border: none;
+  .nx-tree-view__twisty {
+    display: none;
   }
 }
 
 .nx-tree-view__trigger {
   @include container-horizontal;
   align-items: center;
-  border: 1px solid transparent;
+  background: none;
+  border: none;
+  box-sizing: border-box;
+  color: #2c2c2c;
   cursor: pointer;
   display: flex;
-  height: 30px;
-  line-height: 32px;
+  font: inherit;
+  font-weight: 600;
```

Comment:
Why not use `@include: semi-bold();`?

Reply:
> Why not use @include: semi-bold();?

Ah, right.  Generally I avoid that as just setting the `font-weight` is just as easy, but in this case it makes sense since we have to reset the font face as well.  622b3422

Reply:
> Ah, right. Generally I avoid that as just setting the font-weight is just as easy, but in this case it makes sense since we have to reset the font face as well. 622b342

I've had to revert this.  The thing is, this element is a `<button>` and has all sorts of defaults set, including `font-size`.  The mixin doesn't affect `font-size`.  To me it seems easier to reset all of the font properties with `font: inherit` and then customize the weight. be9f758c